pkg/plugin/v3/scaffolds/internal/templates/config/api/webhook_suitetest.go

```diff
@@ -0,0 +1,223 @@
+package api
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"sigs.k8s.io/kubebuilder/pkg/model/file"
+)
+
+var _ file.Template = &WebhookSuite{}
+var _ file.Inserter = &WebhookSuite{}
+
+// WebhookSuite scaffolds the webhook_suite.go file to setup the webhook test
+type WebhookSuite struct {
+	file.TemplateMixin
+	file.MultiGroupMixin
+	file.BoilerplateMixin
+	file.ResourceMixin
+
+	// BaseDirectoryRelativePath define the Path for the base directory when it is multigroup
+	BaseDirectoryRelativePath string
+}
+
+// SetTemplateDefaults implements file.Template
+func (f *WebhookSuite) SetTemplateDefaults() error {
+	if f.Path == "" {
+		if f.MultiGroup {
+			if f.Resource.Group != "" {
+				f.Path = filepath.Join("apis", "%[group]", "%[version]", "webhook_suite_test.go")
+			} else {
+				f.Path = filepath.Join("apis", "%[version]", "webhook_suite_test.go")
+			}
+		} else {
+			f.Path = filepath.Join("api", "%[version]", "webhook_suite_test.go")
+		}
+	}
+	f.Path = f.Resource.Replacer().Replace(f.Path)
+
+	f.TemplateBody = fmt.Sprintf(webhookTestSuiteTemplate,
+		file.NewMarkerFor(f.Path, importMarker),
+		file.NewMarkerFor(f.Path, addSchemeMarker),
+		file.NewMarkerFor(f.Path, addWebhookManagerMarker),
```

Comment:
@camilamacedo86 To make it work, I have to add two strings "%s", "%d" or "%v" here, so that those get replaced at the time of scaffolding... Should I do that? WDYT?